rotkehlchen/db/upgrades/v43_v44.py

```diff
@@ -96,6 +98,25 @@ def _addressbook_schema_update(write_cursor: 'DBCursor') -> None:
     fix_address_book_duplications(write_cursor=write_cursor)
 
 
+@enter_exit_debug_log()
+def _add_uniswap_to_historical_oracles(cursor: 'DBCursor') -> None:
+    """Add uniswapv2 and uniswapv3 to the list of historical price oracles."""
+    cursor.execute('SELECT value FROM settings WHERE name=?', ('historical_price_oracles',))
+    price_oracles = cursor.fetchone()
+
+    try:
+        oracles = json.loads(price_oracles[0] if price_oracles else '')
```

Comment:
I just notice that this will fail. You are doing an append later so it should be a list